components/family-studies/ui/src/main/resources/PhenoTips/FamilyRecordField_externalId.xml

```diff
@@ -108,16 +108,10 @@
       </scope>
     </class>
     <property>
-      <content>{{include reference="PhenoTips.FormMacros" /}}
+      <content>{{include reference="PhenoTips.PatientSheetMacros" /}}
 
 {{velocity}}
-#if($xcontext.action == 'edit' || $doc.display('external_id') != '')
-  (% class="fieldset external_id" %)(((
-    #__form_label('external_id' 'section')
-
-    #__form_field('external_id')
-  )))
-#end
+#__displayFormatted('2-col' 'external_id' '' '' '')
```

Comment:
Why this change? The 2 column layout doesn't look well in the family page, please see https://github.com/phenotips/phenotips/commit/9e4d44967232e0ff4cfecabe6d4d8679497b4ea8#diff-67005384f8134222422f15ae2a7ac382 .